packages/utilities/psammead-locales/moment/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  extends: [
+    "eslint:recommended",
+    "plugin:es5/no-es2015"
```

Comment:
The documentation describes this as "Forbid ES2015 usage". That doesn't seem right?

Reply:
ES2015 Is ES6 so this is actually right